pkg/cmd/attestation/verify/verify.go

```diff
@@ -52,10 +52,15 @@ func NewVerifyCmd(f *cmdutil.Factory, runF func(*Options) error) *cobra.Command
 			The %[1]s--owner%[1]s flag value must match the name of the GitHub organization
 			that the artifact's linked repository belongs to.
 
-			By default, the verify command will attempt to fetch attestations associated
-			with the provided artifact from the GitHub API. If you would prefer to verify
-			the artifact using attestations stored on disk (c.f. the %[1]sdownload%[1]s command),
-			provide a path to the %[1]s--bundle%[1]s flag.
+			By default, the verify command will:
+			- only verify provenance attestations
+			- attempt to fetch relevant attestations via the GitHub API.
+
+			To verify other types of attestations, use the predicate-type flag.
```

Comment:
When we mention other flags, we're consistent in bolding the text:

```suggestion
			To verify other types of attestations, use the %[1]s--predicate-type%[1]s flag.
```

ditto below for the `--bundle-from-oci` flag, the `--bundle` flag and the `download` command 🙂.